docs/source/conf.py

```diff
@@ -9,10 +9,35 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("."))
+
+print(sys.executable)
+
+if os.environ.get("READTHEDOCS") == "True":
+    # we need to add ffmpeg to the path
+    venv_path = os.environ.get("VIRTUAL_ENV")
+    ffmpeg_path = os.path.join(
+        venv_path, 
+        "lib",
+        "python3.8",
+        "site-packages",
+        "imageio_ffmpeg",
+        "binaries"
+    )
+    # the included binary is named ffmpeg-linux..., create a symlink
+    [ffmpeg_bin] = [file for file in os.listdir(ffmpeg_path) if file.startswith("ffmpeg-")]
+    os.symlink(
+        os.path.join(ffmpeg_path, ffmpeg_bin), 
+        os.path.join(ffmpeg_path, "ffmpeg")
+    )
+    os.environ["PATH"] += os.pathsep + ffmpeg_path
+
```

Comment:
@behackl Did this work?

Reply:
No, but I am about to try something else that is more likely to work.